next.config.js

```diff
@@ -499,88 +499,88 @@ async function redirects() {
     },
     {
       source: '/naming-services/build-profile-search-index',
-      destination: '/technology/naming-system/',
```

Comment:
I mainly mean that, if I'm interpreting this correctly, `/technology/naming-system` will now be a broken route. We need to set a precedent that if you change a route, you have to make sure it still works.

Reply:
By changing a `permanent` route, anyone who has visited `/naming-services/build-profile-search-index` will automatically go to `/technology/naming-system`, because it'll be in their browser cache. So, I think we need to be creating _2_ new redirects.

Reply:
Yep I still need to add a bunch of redirects for the changed pages here 👍